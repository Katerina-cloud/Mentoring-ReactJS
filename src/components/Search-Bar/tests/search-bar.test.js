import React from 'react';
import { render } from '@testing-library/react';
import { SearchBar } from '../';

describe('SearchBar', () => {
  it('snapshot', () => {
    const tree = render(<SearchBar />);
    expect(tree).toMatchSnapshot();
  });
});
