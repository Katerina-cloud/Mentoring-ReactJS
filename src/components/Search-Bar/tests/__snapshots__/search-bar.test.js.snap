// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchBar snapshot 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="search-bar"
      >
        <h1
          class="search-bar__title"
        >
          Find your movie
        </h1>
        <div
          class="search-bar__search-container"
        >
          <div
            class="search-bar__form-input"
          >
            <input
              class="form-input__text"
              placeholder="What do you want to watch?"
              value=""
            />
          </div>
          <div
            class="search-bar__search-button"
          >
            <button
              class="button button--text-undefined button--red button--border-undefined"
            >
              Search
            </button>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="search-bar"
    >
      <h1
        class="search-bar__title"
      >
        Find your movie
      </h1>
      <div
        class="search-bar__search-container"
      >
        <div
          class="search-bar__form-input"
        >
          <input
            class="form-input__text"
            placeholder="What do you want to watch?"
            value=""
          />
        </div>
        <div
          class="search-bar__search-button"
        >
          <button
            class="button button--text-undefined button--red button--border-undefined"
          >
            Search
          </button>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
